cogs5e/dice.py

```diff
@@ -31,31 +31,60 @@ async def quick_roll(self, ctx, *, mod: str = '0'):
 
     @commands.command(name='roll', aliases=['r'])
     async def rollCmd(self, ctx, *, rollStr: str = '1d20'):
-        """Rolls dice in xdy format.
+        """Roll is used to roll any combination of dice in the `XdY` format. (`1d6`, `2d8`, etc)
+        
+        Multiple rolls can be added together as an equation. Standard Math operators and Parentheses can be used: `() + - / \*`
```

Comment:
```suggestion
        Multiple rolls can be added together as an equation. Standard Math operators and Parentheses can be used: `() + - / *`
```
Looks like that backslash is unnecessary since it's already in a code block